build/docker/Dockerfile

```diff
@@ -41,11 +41,6 @@ ENV GITHUB_CLI_VERSION 1.14.0
 RUN curl --silent --location "https://github.com/cli/cli/releases/download/v${GITHUB_CLI_VERSION}/gh_${GITHUB_CLI_VERSION}_linux_amd64.tar.gz" \
     | tar xvz -C /out/usr/local/bin --strip-components=2 gh_${GITHUB_CLI_VERSION}_linux_amd64/bin/gh && gh --version
 
-# Flux release very regularly so we always get and test against the latest
```

Comment:
Okay, so... I thought this would trigger the need to update the build image? But when I run:

```
make -f Makefile.docker update-build-image-tag
```

nothing happens... am I missing something?